src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -294,6 +267,7 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
 
   /** Creates the scenario and the uploaded project areas, if provided. */
   createScenarioAndProjectAreas(): void {
```

Comment:
nit since this was already here - this is certainly creating a scenario, but it is not quite creating project areas yet.

Reply:
I believe the projectArea creation here is for uploaded project areas (which isn't implemented now, but may be in the future). Since we're not doing that now, I can just change the name and add a TODO to add support for uploading project areas later

Reply:
ack, got it.